### ISSUE TITLE
Help with developing plugins

### ISSUE THREAD

**Issue by dczaretsky** [revive-adserver/revive-adserver#694]
Hello,
I've developed a few plugins previously for revive, but now looking to add some new functionality to campaigns and banners. I can't seem to find any documentation on how to build a mod for the current campaign or banner pages (create / edit).  Can someone provide some direction, or perhaps a template that I can use. 

On a side note, I believe the reason why there such poor community development for revive is that documentation is non-existent. We really need more than a handful of blogs and outdated tutorials that don't dig deep into the code and algorithms. There are only a few companies building plugins, and I think it would benefit all if there was a better approach to getting the community involved. Just look at Drupal as an example of how communities can work together to build and test frameworks together. Just sayin...

**Comment by erikgeurts** [comment]
I would agree that it would be great if more people started developing plugins for Revive Adserver, and I also agree that it would be nice to have some more documentation on that.

Revive Adserver is being maintained by a small team of developers. Ad serving is a very specialized topic. It is not a fair comparison to point to Drupal. Since there is a much bigger market for CMS applications, they have a much larger community.

I think the best place to continue this discussion on our community forums, in this case at http://forum.revive-adserver.com/forum/22-plugins/ , so that we can keep our Github issues list restricted to bugs and other code issues.
